app/views/devise/registrations/new.html.erb

```diff
@@ -16,9 +16,17 @@
             <div class="card-body">
               <div class="m-sm-4">
                 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+                <div class="mb-3">
+                  <label>First Name</label>
+                  <%= text_field_tag :firstname, nil, autofocus: true, class:'form-control' %>
+                </div>
+                <div class="mb-3">
+                  <label>Last Name</label>
+                  <%= text_field_tag :lastname, nil, class:'form-control' %>
+                </div>
```

Comment:
Even though internally, we use "firstname" and "lastname", in the interfaces we usually use "Given Name" and "Family Name".